diagnostics/stratonovich_additive.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import tqdm
+from scipy import stats
+
+from tests.basic_sde import AdditiveSDE
+from torchsde import sdeint, BrownianInterval
+from torchsde.settings import LEVY_AREA_APPROXIMATIONS
+from .utils import to_numpy, makedirs_if_not_found, compute_mse
+
+from tests.problems import Ex3Additive
+
```

Comment:
Typical style guides enforce two blank lines between last import and first function/class definition.

Reply:
Done!